package.json

```diff
@@ -23,19 +23,19 @@
   "dependencies": {
     "lodash": "^4.0.0",
     "mongoose": "^4.4.11",
-    "trailpack": "^1.0.2",
-    "trailpack-datastore": "^1.0.0",
-    "trails-service": "^1.0.0-beta-2"
+    "trailpack": "2.1.0",
+    "trailpack-datastore": "2.0.0",
+    "trails-service": "2.0.0"
   },
   "devDependencies": {
     "chai": "^3.5.0",
-    "eslint": "^2.10.0",
-    "eslint-config-trails": "^1.0.6",
-    "mocha": "^2.4",
-    "smokesignals": "^1.2.2",
-    "trailpack-core": "^1.0.0-beta-6",
-    "trails": "^1.0.0",
-    "trails-model": "^1.0.0-beta-2"
+    "eslint": "3.13.1",
+    "eslint-config-trails": "2.0.6",
+    "mocha": "3.2.0",
+    "smokesignals": "2.1.0",
+    "trailpack-core": "2.0.0",
+    "trails": "2.0.0",
+    "trails-model": "2.0.1"
```

Comment:
Classes are not needed anymore to, you can remove them and use `require('trails/model')`